cloud/pkg/taskmanager/manager/executor.go

```diff
@@ -208,17 +208,16 @@ func initExecutor(message util.TaskMessage) (*Executor, error) {
 		return nil, err
 	}
 	if len(nodeStatus) == 0 {
-		nodeList := controller.ValidateNode(message)
-		if len(nodeList) == 0 {
-			return nil, fmt.Errorf("no node need to be upgrade")
+		successNodes, failNodes := controller.ValidateNode(message)
+		nodeStatus = make([]v1alpha1.TaskStatus, 0, len(message.Name))
+		for _, node := range failNodes {
+			nodeStatus = append(nodeStatus, v1alpha1.TaskStatus{NodeName: node.Name, State: api.TaskFailed})
```

Comment:
Please set fields Event, Action and Reason to ensure that the error is traceable.

Reply:
Sorry，it passed a week.Now is updated.